### PR TITLE
Added week 7 blog post

### PR DESCRIPTION
**Pull request by m-agour**
Week 7 blog post.

### CODE REVIEW

File: docs/source/posts/2022/2022-08-01-week-7-mohamed.rst [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+What did you do this week?
+--------------------------
+
+- Restructured the keyframe animation interpolators using closures to be functions instead of classes `#647`_. Now it is easier to implement new interpolators with the help of the functions existing in `fury/animation/helpers.py`_. Also, now unit tests can be added for the latter functions.
+
+- Added two examples explaining how to implement a custom interpolator that can be used by the ``Timeline``, one using `classes`_ and the other using `closures`_.
+
+- Fixed rotation issue that Shivam discovered while using the ``Timeline`` to animate glTF models. So, rotation in VTK is done by rotating first around Z-axis, then X-axis, then finally around Y-axis, which was not the order I was using to convert from quaternions to Euler degrees.
+
+- Made changes requested by Javier and Filibi on the billboards using geometry shader `PR`_, and made an example of how to use this billboard to show an impostor sphere which looks almost like a real high poly sphere. Also benchmarked using this version of billboard vs using a quad-based billboard, and how they both affect the FPS of the animation.
```
Comment (ganimtron-10) [inline docs/source/posts/2022/2022-08-01-week-7-mohamed.rst:19]: `Filipi` instead

File: docs/source/posts/2022/2022-08-01-week-7-mohamed.rst [hunk 2]
```diff
@@ -0,0 +1,40 @@
+Week 7: Billboard spheres and implementing interpolators using closures
+==========================================================
```
Comment (ganimtron-10) [inline docs/source/posts/2022/2022-08-01-week-7-mohamed.rst:2]: Title underline too short

**Review by ganimtron-10 — COMMENTED** [review]
Hello @m-agour ,
Good Blog! 
PTAL at below comments.
Thanks!

**Comment by skoudoro** [comment]
@xtanion ???

**Review by skoudoro — COMMENTED** [review]
LGTM, merging. thanks !

**Comment by xtanion** [comment]
Apologies for the late response

**Pull request merged**